CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 3.0.1, 2023-10-24
```

Comment:
I believe this calls for 3.1.0. just in case, since it's not just a patch but refactoring of how things work.

Reply:
I do agree.

Reply:
> this calls for 3.1.0

Makes sense. Changed.